source/nodejs/.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
Does it work on both windows and iOS? after run the commit command. will it still in / other than /source/nodejs?

Reply:
works on Windows, iOS, and Linux. good point about running when there aren't files modified in `source/nodejs` - I added a little pre-filter here: 9971adb7386f0bc2fe8cbcbe50e9bb0057d4e36a